### DISCUSSION TITLE
Using Quartznet for WPF application with Target Framework .Net 6

### DISCUSSION THREAD

**Discussion by VijrumbhanM** [quartznet/quartznet#1541] [Q&A]
Hi Team,

my application is .net framework 4.8 targeted, and i am planning to migrate to latest .Net 6. 
so i have upgraded my application using microsoft upgrade assistant. 
as the upgrade analysis HttpContext.Current and some other features used by quartz are not supported in new framework.

i would want to know if we can use quartz in  .net6 application is it compatible. 

what should we do to continue using quartz in my .net 6 wpf application

**Comment by lahma** [comment]
Yes, Quartz .is NET 6.0 compatible.
